src/main/java/Parser.java

```diff
@@ -0,0 +1,74 @@
+import exception.EmptyDescriptionException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+    private String input;
+    private String command;
+    private String[] inputs;
+
+    public Parser(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Returns the command such as "todo", "deadline", "event".
+     * @return String command.
+     */
+    public String getCommand() {
+        inputs = input.split(" ");
+        command = inputs[0];
+        return command;
+    }
+
+    /**
+     * Gets the task depending on the command.
+     * @return String representing task.
+     */
+    public String getTask() {
+        switch (command) {
+        case "todo":
+            return inputs[1];
+        case "find":
+            return inputs[1];
+        case "deadline":
+            return input.split(" /by")[0].split(" ", 2)[1];
+        case "event":
+            return input.split(" /at")[0].split(" ", 2)[1];
+        default:
+            return "";
+        }
+    }
+
+    /**
+     * Gets the date depending on whether task is "deadline" or "event".
+     * @return LocalDateTime object.
```

Comment:
Try to leave an empty line between the parameter descriptions and method description.

Reply:
Yes! Thank you